hack/arktos-up.sh

```diff
@@ -396,11 +396,14 @@ Alternatively, you can write to the default kubeconfig:
 
   export KUBERNETES_PROVIDER=local
 
-  cluster/kubectl.sh config set-cluster local --server=https://${API_HOST}:${API_SECURE_PORT} --certificate-authority=${ROOT_CA_FILE}
-  cluster/kubectl.sh config set-credentials myself ${AUTH_ARGS}
-  cluster/kubectl.sh config set-context local --cluster=local --user=myself
-  cluster/kubectl.sh config use-context local
+  sudo cluster/kubectl.sh config set-cluster local --server=https://${API_HOST}:${API_SECURE_PORT} --certificate-authority=${ROOT_CA_FILE}
```

Comment:
We don't use sudo for kubectl. This should not be a hard requirement

Reply:
The new commits remove 'sudo' permission. The procedure can run smoothly without sudo permission.